src/transformers/tokenization_utils.py

```diff
@@ -135,6 +255,7 @@ def __init__(self, **kwargs):
         self.added_tokens_encoder: Dict[str, int] = {}
         self.added_tokens_decoder: Dict[int, str] = {}
         self.unique_no_split_tokens: List[str] = []
+        self.trie = Trie()
```

Comment:
Would it be possible to have a more explicit name of the property? For example `unique_tokens_trie`?

Reply:
I don't like `unique` in there, because there's nothing related to uniqueness in there.

`no_split_trie` could work, but we're actually using it to split around some tokens.
`tokens_trie` could work IMO.
`no_split_tokens_trie` is a bit longer and not really informative (there are no `split_tokens` afaik).